resources/public/api/conf/2.0/schemas/retrieveBalanceAndTransactions/itemDetails.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Retrieve History of a Self Assessment Charge",
+  "description": "This endpoint enables you to retrieve the history of changes to a single charge made by HMRC. A National Insurance number and transaction ID must be provided.",
+  "type": "object",
+  "properties": {
+    "items": {
+      "type": "array",
+      "description": "The items Schema",
+      "items": {
+        "type": "object",
+        "description": "The list of items as part of the financial details",
+        "properties": {
+          "subItem": {
+            "description": "An identifier to keep track of the number of items in this list",
+            "type": "string",
+            "example": "001"
+          },
+          "dueDate": {
+            "description": "If the requested item relates to a charge, the due date is the date by which the amount is due to be paid. If the requested item relates to a payment, the due date refers to the date the payment was made to HMRC in the format YYY-MM-DD",
+            "type": "string",
+            "example": "2018-04-05"
+          },
+          "amount": {
+            "description": "The amount that this entry changed the transaction by. The value must be between -99999999999.99 and 99999999999.99 to 2 decimal places.",
+            "type": "number",
+            "multipleOf": 0.01,
+            "minimum": -99999999999.99,
+            "maximum": 99999999999.99,
+            "example": "5009.99"
+          },
+          "clearingDate": {
+            "description": "The date this entry changed the item, in the format YYYY-MM-DD.",
+            "type": "string",
+            "example": "2018-04-05"
+          },
+          "clearingReason": {
+            "description": "The reason for the change to the item.",
+            "type": "string",
+            "example": "01"
+          },
+          "outgoingPaymentMethod": {
+            "description": "The method used to make the outgoing payment. One of Repayment to Card, Payable Order Repayment or BACS Payment out",
+            "example": "Payable Order Repayment"
```

Comment:
"enum" :[
              "Repayment to Card",
              "Payable Order Repayment",
              "BACS Payment out"
            ]
is missing